toffy/test_utils.py

```diff
@@ -300,6 +299,11 @@ def case_default(self, intensity):
             check_qc_dir_structure,
         ]
         return [
-            functools.partial(build_extract_callback, panel=panel, intensities=intensity),
-            functools.partial(build_qc_callback, panel=panel),
+            functools.partial(
+                build_fov_callback,
+                'extract_tiffs',
+                'generate_qc',
+                panel=panel,
```

Comment:
These functions could use some comments

Reply:
just the watcher cases or the whole suite of watcher/callback test utils?

Reply:
Mostly the cases, hard to interpret what's going on with the partial function building, especially looking forward for adding in additional callbacks